README.md

```diff
@@ -49,240 +24,151 @@ After you're finished please remove all the comments and instructions!
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
+# 📖 [Country-Information] <a name="about-project"></a>
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+\*\*[Country-Information] is a mobile app that consumes an external API to show details about most country in the world. users are able to search for any specific country they wish to find information from. Each country comes pre-poulated with its name and its population. In more details, there are information like capital city, area, timezone, and even the continent the country resides.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
+  <summary>React</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://react.dev/">React</a></li>
   </ul>
 </details>
 
-<details>
-  <summary>Server</summary>
+  <summary>JavaScript</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
   </ul>
 </details>
 
-<details>
-<summary>Database</summary>
+  <summary>HTML</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">html</a></li>
   </ul>
 </details>
 
-<!-- Features -->
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">css</a></li>
+  </ul>
+</details>
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[React]**
+- **[React Hooks]**
+- **[React API]**
+- **[React Routers]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
+## 🚀 Live Demo <a name="live-demo" ></a>
 
-> Add a link to your deployed project.
+- [See Live](https://country-information-e4bl.onrender.com)
 
-- [Live Demo Link](https://google.com)
+- [Video Link](https://www.loom.com/share/48d8c4ebcba94694ba6aaf87179d9895)
```

Comment:
- [ ] [OPTIONAL] A great presentation you have here @Adholah96. However, I will like to recommend that yus show up some parts of your code next time because you stated that the part of the project you are most proud of is the test, mock tests. Also, you will be glad to keep your camera on whenever you are making a presentation, these are all professional ways of making a presentation.

Reply:
Hello @Whoistolu this is duely noted. Thank you so much for your feedback and bare in min that I have taken everything and will improve on them.